eos-tech-support/eos-stage-flatpak

```diff
@@ -69,6 +69,23 @@ for remote in `flatpak remote-list --system | awk {'print $1'}` ; do
         echo "Setting flatpak $remote URL to $new_url"
         flatpak remote-modify "$remote" --url="$new_url"
 
+        # Update the collection ID if it's set
+        collection_id=$(ostree --repo="$flatpak_repo" config get 'remote "'"$remote"'".collection-id' 2>/dev/null || echo "")
+        if [ -n "$collection_id" ] ; then
+            if [ "$STAGE" == "prod" ] ; then
+                new_collection_id=`echo "$collection_id" | sed -e "s/Dev.//" | sed -e "s/Demo.//"`
```

Comment:
Is it guaranteed that there is something after the ".Dev"?  If not, this won't quite work.  For example...
```
roddy@endless:~/checkout/glib$ echo com.endlessm.Dev | sed "s/Dev.//"
com.endlessm.Dev
```

Reply:
Yes, it's guaranteed

Reply:
Could do with some examples of the formats of collection ID which this will deal with, in the comment above the `sed` line.